transform/mattermost-analytics/models/staging/mattermostcom/_mattermostcom__models.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+
+models:
+  - name: stg_mattermostcom__tracks
+    description: |
+      Reconstructed `tracks` table using custom properties expected to be in the events.
+
+    columns:
+      - name: event_id
+        description: The event's id.
+      - name: event_name
```

Comment:
@ifoukarakis , flipping the description texts?

Reply:
Checked the data and seems event name is always `feedback_submitted`. Updating description.

Reply:
@ifoukarakis , I think you are referring to one of the other PRs comments. I think here it is just reversing `The name of the event table.` with `The name of the event.`

Reply:
Oops, correct. Descriptions were switched here. Just left the comment here rather than in the other PR.